packages/node_modules/@ciscospark/redux-module-conversation/src/reducer.js

```diff
@@ -65,17 +58,18 @@ export default function reducer(state = initialState, action) {
     let activities = new OrderedMap(conversationActivities.map((activity) => [activity.url, activity]));
     activities = activities.sortBy((activity) => activity.published);
 
-    return Object.assign({}, state, {
-      activities,
-      defaultActivityEncryptionKeyUrl,
-      id,
-      kmsResourceObjectUrl,
-      locusUrl,
-      url,
-      isFetching: false,
-      isLoaded: true,
-      participants: participants.items
-    });
+    return state
+      .set(`activities`, activities)
+      .merge({
+        defaultActivityEncryptionKeyUrl,
```

Comment:
Should we put the conversation object that the sdk is expecting into its own property of the reducer? That way we can do `conversation.conversationObject.toJS()` when calling sdk methods. It might speed up the `toJS` methods since we won't have to convert all of the activities every time.